other/js/websockify.js

```diff
@@ -92,7 +93,7 @@ http_request = function (request, response) {
 
     var uri = url.parse(request.url).pathname
         , filename = path.join(argv.web, uri);
-    
+
```

Comment:
This white space change is not really close to other modified code. I would prefer if you could move it to a separate commit